### DISCUSSION TITLE
Get an error in Angular 10.2.4 when installing angular-datatables 10.1.2

### DISCUSSION THREAD

**Discussion by federicopenaranda** [l-lin/angular-datatables#1526] [Q&A] [answered]
Hi, I'm upgrating an Angular app to version 10.2.4, I've solved every issue, but I get this error:

```
ERROR in src/app/app.module.ts:13:34 - error TS2307: Cannot find module 'angular-datatables' or its corresponding type declarations.

13 import { DataTablesModule } from "angular-datatables";
```
It seems that it doesn't recognize the package "angular-datatables", in Angular version 9 it was working fine.

I've tried everything, removing node_modules, and downloading it again, I've downgraded angular-datatables, I've reinstalled "datatables.net" and all other dependencies, I've used "ng add angular-datatables", I've installed everything again manually with all the npm install ... in the docs, and I can't fix the problem, I attach the package.json file so you can see my environment, I'm using Ubuntu 20.04. Your help would be very appreciated, thank you.

**Comment by shanmukhateja** [comment, marked as answer]
Hi, could you provide the exact version numbers for Angular, NodeJS, Angular CLI as well as angular-datatables?

What happens if you remove `node_modules/angular-datatables` and run `ng add angular-datatables@10.1.2`. If this fails, repeat the steps with `ng add angular-datatables@10.1.1`

**Reply by federicopenaranda** [reply]
Hi shanmukhateja, thank you for your response, adding angular-datatables 10.1.1 did the trick!. Anyway, I'm attaching my environment data for future reference, for you and for others that might have the same issue.

I'm attaching the package.json file (which I failed attaching in my first post!), these are the versions I working with:
- Angular CLI: v10.2.3 (I realize that I was working with an old version of Angular CLI -> 6.2.0!)
- Angular: v10.2.4
- NodeJS:  v14.15.5
- angular-datatables: 10.1.2

Thank you very much!!!

[package.txt](https://github.com/l-lin/angular-datatables/files/6180730/package.txt)
